apps/src/templates/TimeAgo.jsx

```diff
@@ -13,6 +13,38 @@ class TimeAgo extends React.Component {
     style: PropTypes.object
   };
 
+  state = {
+    timeout: undefined
+  };
+
+  componentDidMount() {
+    this.startTimer();
+  }
+
+  componentWillUnmount() {
+    this.clearTimer();
+  }
+
+  startTimer = () => {
+    // Update the display string once a minute.
+    //
+    // Note this component could be much more clever; if the given date is
+    // within a few minutes of now, update this quickly, but if it's a couple
+    // hours ago update much less frequently and if it's even longer ago then
+    // maybe don't bother updating at all.
+    //
+    // For now, that seems like a unnecessary optimization.
+    const timeout = setTimeout(this.startTimer, 60 * 1000);
```

Comment:
I _do_ think we care about the sub-one-minute updates when they first make a change, because it's the only way for a user to see if their most recent changes have been saved. If it says "saved just now" for up to a minute, that's enough time for me to make an edit, think it has saved, and navigate away — possibly losing work.

Reply:
As discussed offline, the only initial change we've ever had here has been from "just now" to "a minute ago"; the current setup preserves that.